app/policies/loa3_user_policy.rb

```diff
@@ -0,0 +1,26 @@
+class Loa3UserPolicy
```

Comment:
I'm not sure I understand the naming and use of these, especially since we are initializing 2 different policy objects for the same resource in the controller.

Why does there need to be 2 policy objects. It seems like we could have one PersonalKeyConfirmationPolicy object that handles this for both cases.

Reply:
Or maybe `UserSigningUpPolicy` with methods `via_loa3?` and `not_via_loa3?`?